benchmarks/tpcc/src/main/java/com/google/cloud/pgadapter/tpcc/SchemaService.java

```diff
@@ -44,21 +46,24 @@ void createSchema() throws IOException, SQLException {
               .executeQuery(
                   "select count(1) "
                       + "from information_schema.tables "
-                      + "where table_schema='public' "
-                      + "and table_name in ('warehouse', 'district', 'customer', 'history', 'orders', 'new_orders', 'order_line', 'stock', 'item')")) {
+                      + "where "
+                      + (is_googlesql ? "" : "table_schema='public' and ")
```

Comment:
This could break once we introduce named schema support.

```suggestion
                      + (is_googlesql ? "table_schema='' and " : "table_schema='public' and ")
```

Reply:
Good catch. Fixed.